corgi/core/models.py

```diff
@@ -978,7 +978,7 @@ def get_source(self) -> list:
         return list(
             ComponentNode.objects.get_queryset()  # type: ignore
             .filter(purl=self.purl)
-            .get_ancestors(include_self=True)
+            .get_ancestors(include_self=False)
```

Comment:
I think include_self=False is the default, so up to you whether you just remove the kwarg, or leave it here to be more explicit.

Reply:
I'd prefer to keep it explicit.